### ISSUE TITLE
[Question] Loading android shared library

### ISSUE THREAD

**Issue by y3l1n4ung** [TimWhiting/dartpy#6]
Due to this [stackoverflow q/a](https://stackoverflow.com/questions/10881411/load-native-shared-library-from-another-android-application) , I think , I can also load lib from Termux and use in android.

**Comment by TimWhiting** [comment]
You are wanting to use this in plain dart on Android then? No flutter?

**Comment by y3l1n4ung** [comment]
No, I just want to load python lib from termux for my flutter app.
Eg. 
1.install python on termux 
2.import python lib from termux just like in Linux and Window
3.and use in flutter app with your package
I think you understand that.
As of now, I copy android python lib from assets 
but when I run python command it show "import error "
because I think python library or path need to be set (You advise).

**Comment by TimWhiting** [comment]
Yes the PYTHONPATH needs to be set.
